src/control/lib/support/log.go

```diff
@@ -62,11 +63,10 @@ type LogTypeSubCmd struct {
 }
 
 const (
-	MMDDYYYY        = "1-2-2006"
-	HHMMSS          = "15:4:5"
-	MMDDHHMMSS      = "1/2-15:4:5"
-	MMDDYYYY_HHMMSS = "1-2-2006 15:4:5"
-	YYYYMMDD_HHMMSS = "2006/1/2 15:4:5"
+	YYYYMMDD            = "2006-01-02"        // Date format as it is expected from commandline argument
+	HHMMSS              = "15:4:5"            // Time format as it is expected from commandline argument
+	YYYYMMDD_HHMMSS     = "2006-01-02 15:4:5" // Date/Time format as it is defined by ISO 8601
+	YYYYMMDD_HHMMSS_LOG = "2006/01/02 15:4:5" // Date/Time format as it is used in DAOS logs
```

Comment:
Huh, does the time format need to be corrected too? I.e. `15:04:05`?

Reply:
Yes, thank you Kris, you are right.
Fixed in https://github.com/daos-stack/daos/pull/17128/commits/78c3ac36f935b557389fa7490834999a4d714b62